### PR TITLE
Feat: CU-865c60xaf: recover password

### PR DESCRIPTION
**Pull request by raulynzunza**
# Fixes ticket [#CU-865c60xaf](https://app.clickup.com/t/CU-865c60xaf)

## Problem Description
We needed a Recovery Password Form

## Checklist 
- [x] Passes eslint linter 
- [ ] Passes backend integration tests 
- [ ] Added tests for this change 
- [ ] Added documentation for this change 
- [ ] Removed unused packages  

## Changes proposed in this pull request
Added a form that allows to recover your password

## Reviewers
- @jorgelabs 
- @JoHena 
- @aviada-rob 
- @aviada-andreazayas

### CODE REVIEW

**Review by JoHena — APPROVED** [review]
LGTM!

File: src/app/components/Organisms/ResetForm/ResetForm.tsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
+import TextboxInput from "../../Atoms/TextboxInput/TextboxInput";
+import BaseButton from "../../Atoms/Buttons/BaseButton/BaseButton";
+
+const ResetForm: React.FC = () => {
```
Comment (jorgelabs) [inline src/app/components/Organisms/ResetForm/ResetForm.tsx:4]: Consider renaming "ResetForm" to a more descriptive name, such as "PasswordResetForm" or "ForgotPasswordForm":

File: src/app/components/Organisms/ResetForm/ResetForm.tsx [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  return (
+    <>
```
Comment (jorgelabs) [inline src/app/components/Organisms/ResetForm/ResetForm.tsx:6]: Consider removing the unnecessary fragment <></> in the first line:

File: src/app/components/Organisms/ResetForm/ResetForm.tsx [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+      <form className="w-[21.938rem]">
+        <h5 className="font-serif text-secondary font-bold text-[1.625rem] mb-[1rem]">
+          Recover your password
+        </h5>
+        <p className="text-[0.875rem] mb-4 pb-[0.188rem] text-secondary">
+          Enter your email and you will receive a link to recover your password
+        </p>
+        <TextboxInput
+          icon
+          iconColor="text-blue-primary"
```
Comment (jorgelabs) [inline src/app/components/Organisms/ResetForm/ResetForm.tsx:16]: Consider changing "iconcolor" to "iconColor" for consistency with camelCase convention:

File: src/app/components/Organisms/ResetForm/ResetForm.tsx [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+          iconSize="text-l"
```
Comment (jorgelabs) [inline src/app/components/Organisms/ResetForm/ResetForm.tsx:17]: Consider changing "iconsize" to "iconSize" for consistency with camelCase convention:

File: src/app/components/Organisms/ResetForm/ResetForm.tsx [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+          textboxPlaceholder="placeholder-secondary"
+          textboxSize="text-sm"
+          textboxWidth="w-full"
+          viewBox="0 0 24 24"
+          inputType="email"
+        />
+        <div className="flex justify-end mb-[1.188rem] mt-[1rem]">
+          <BaseButton
+            icon=""
+            iconcolor="text-light"
```
Comment (jorgelabs) [inline src/app/components/Organisms/ResetForm/ResetForm.tsx:32]: Consider changing "iconcolor" to "iconColor" for consistency with camelCase convention

File: src/app/components/Organisms/ResetForm/ResetForm.tsx [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+          Enter your email and you will receive a link to recover your password
+        </p>
+        <TextboxInput
+          icon
+          iconColor="text-blue-primary"
+          iconSize="text-l"
+          iconType="email"
+          margin="m-0"
+          name="Email"
+          personalize=" pointer-events-none absolute top-1/2 right-1 transform -translate-y-2/4"
```
Comment (jorgelabs) [inline src/app/components/Organisms/ResetForm/ResetForm.tsx:21]: Consider separating the logic of the style into a CSS file or an object of styles and passing it to the component via props

File: src/app/components/Organisms/ResetForm/ResetForm.tsx [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+          textboxSize="text-sm"
+          textboxWidth="w-full"
+          viewBox="0 0 24 24"
+          inputType="email"
+        />
+        <div className="flex justify-end mb-[1.188rem] mt-[1rem]">
+          <BaseButton
+            icon=""
+            iconcolor="text-light"
+            iconsize="text-xs"
```
Comment (jorgelabs) [inline src/app/components/Organisms/ResetForm/ResetForm.tsx:33]: Consider changing "iconsize" to "iconSize" for consistency with camelCase convention

File: src/app/components/Organisms/ResetForm/ResetForm.tsx [hunk 8]
```diff
@@ -0,0 +1,51 @@
...
+            marginleft="ml-1"
+            onclickProp={() => {}}
+            personalize=""
```
Comment (jorgelabs) [inline src/app/components/Organisms/ResetForm/ResetForm.tsx:36]: Consider removing the unnecessary "personalize" prop or remove if not used

**Review by jorgelabs — CHANGES_REQUESTED** [review]

**Review by jorgelabs — APPROVED** [review]

**Pull request merged**